### DISCUSSION TITLE
Scheduler error alert

### DISCUSSION THREAD

**Discussion by nsqa99** [kagkarlsson/db-scheduler#409] [Q&A] [answered]
Hi, how can I notice if the Scheduler is down due to some error such as DB disconnection?
I've checked the source code and it seems nothing happened to notify developers when the DB is down.
Thanks!

**Comment by kagkarlsson** [comment, marked as answer]
Nothing direct no. You can monitor the `DataSource` yourself if you wish. 
You might be able to do some error-detection via a custom `StatsRegistry`. If there are no events of type `RAN_EXECUTE_DUE` for a long while, you might be having db-issues.
